benchmarks/filtsmooth.py

```diff
@@ -56,7 +14,8 @@ class Filtering:
     params = [[("ekf", "classic"), ("ekf", "sqrt"), ("ukf", "classic")]]
```

Comment:
for the sqrt implementation to be meaningful, I suppose the problem classes need an `implementation` argument (maybe `forward_implementation` as well as `backward_implementation`). Otherwise cholesky factors keep getting computed and thrown away after each step (unreasonably), which makes benchmarks a bit meaningless.